### PR TITLE
Literate reproducing Taylor coefficients for conformal map

### PR DESCRIPTION
**Pull request by navidcy**
Thought I literate the `compute_Taylor_coefficients.jl` otherwise it gets lost in the sand..

related to #15

### CODE REVIEW

**Comment by Sbozzolo** [comment]
Minor nitpick, but I find that the progress bar doesn't look great when rendered in a tutorial:
https://clima.github.io/CubedSphere.jl/previews/PR40/literated/compute_taylor_coefficients/

(Also, squash commits before merging)

**Comment by navidcy** [comment]
> Minor nitpick, but I find that the progress bar doesn't look great when rendered in a tutorial: https://clima.github.io/CubedSphere.jl/previews/PR40/literated/compute_taylor_coefficients/

True -- I dropped it.

> (Also, squash commits before merging)

Agreed; I changed the defaults of the repo.

**Comment by navidcy** [comment]
@Sbozzolo could you have a look?

File: examples/compute_taylor_coefficients.jl [hunk 1]
```diff
@@ -16,8 +43,11 @@ function find_angles(φ)
     return φ′⁻, φ′⁺
 end
 
-# cbrt goes from W to w
-# cbrt′ goes from W′ to w′
+nothing #hide
```
Comment (Sbozzolo) [inline examples/compute_taylor_coefficients.jl:46]: Why is this needed?

Comment (Sbozzolo) [reply examples/compute_taylor_coefficients.jl:46]: (And the other ones below as well)

File: examples/compute_taylor_coefficients.jl [hunk 2]
```diff
@@ -16,8 +43,11 @@ function find_angles(φ)
...
+
+# Next we define the cubic roots and be carefull to choose the appropriate branch.
```
Comment (Sbozzolo) [inline examples/compute_taylor_coefficients.jl:48]:
```suggestion
# Next we define the cubic roots and be careful to choose the appropriate branch.
```

**Comment by Sbozzolo** [comment]
Preview is here: https://clima.github.io/CubedSphere.jl/previews/PR40/literated/compute_taylor_coefficients/

Looks good to me!

File: examples/compute_taylor_coefficients.jl (hunk 1 above)
Comment (navidcy) [reply examples/compute_taylor_coefficients.jl:46]: not really needed
